Documentation~/Blender.md

```diff
@@ -27,3 +27,42 @@ The plugin will send the following properties directly to Unity:
 > Other properties will require **Bake Modifiers** enabled to get consistent look between Blender and Unity.
 > However, this will also cause the loss of mesh properties in Unity as result of the baking.
 
+---
+
+## Bidirectional Sync
+
+Turn on **Auto-Sync** in the plugin panel to apply property changes in Unity back to Blender.  
+This feature is intended for changing Blender's procedural generation parameters from inside Unity, and 
+requires Blender 3.0 or higher.
+
+
+### Editable Properties in Unity
+
+![](images/MeshSyncServerLiveEditProperties.png)
+
+We can set up custom properties in Blender, from which the plugin will synchronize these properties with 
+the MeshSyncServer **GameObject** in Unity, by adding a MeshSyncServerLiveEditProperties component 
+and putting the properties in it.
+
+The supported property types are:
+* Integer
+* Float
+* Integer Array
+* Float Array
+* String (read-only)
+
+### Editing and Synchronizing Meshes from Unity to DCC Tool
+
+We can also edit and send meshes back to Blender if we enable **Use Pro Builder** option in the MeshSyncServer **GameObject** in Unity,
+which will allow us to change geometry node mesh inputs from Unity easily.  
+This option will only appear if we install [ProBuilder](https://docs.unity3d.com/Packages/com.unity.probuilder@5.0/manual/index.html) package in the Unity project, 
```

Comment:
Maybe change this to:
We can also edit and send meshes back to Blender if we enable the **Use Pro Builder** option in the MeshSyncServer **GameObject** in Unity, which will allow us to change geometry node mesh inputs from Unity easily.
This option will only appear if we install [ProBuilder](https://docs.unity3d.com/Packages/com.unity.probuilder@5.0/manual/index.html) package in the Unity project.